### DISCUSSION TITLE
How to run/use different node versions in different terminals at the same time in different projects.

### DISCUSSION THREAD

**Discussion by bnmunesh** [coreybutler/nvm-windows#1152] [Support] [answered]
How to use and run different Node&npm versions in different terminals for different projects at the same time.
Right now if I change the node version in one terminal, the version changes in the other terminal too. How to use specific versions for different project terminals at the same time.

**Comment by coreybutler** [comment, marked as answer]
This project isn't designed for terminal-specific runtimes. It's still possible, though less intuitive, with [this approach](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#how-can-i-use-two-different-versions-of-nodejs-at-the-same-time) though.

**Reply by bnmunesh** [reply]
Thanks, I'll have a read.
